index.d.ts

```diff
@@ -125,6 +127,18 @@ export interface Locale {
 
 export type LocaleCollection = ContentfulCollection<Locale>;
 
+export interface Tag {
+    name: string
+    sys: {
+      id: string
+      type: 'Tag'
+      version: number
+      visibility: string
```

Comment:
Since visibility can only be one of `public` or `private`, better to explicitly use those instead like so `'public' | 'private'`?

Reply:
Maybe just make it `public` - since we only deliver those on the CDA?